.env.template.development

```diff
@@ -0,0 +1,129 @@
+IN_DOCKER=true
+
+# Solr setting
+SOLR_URL_SCHEME=http
+SOLR_HOST=solr
+SOLR_PORT=8983
+SOLR_CORE=hyrax_production
+SOLR_TEST_PORT=8983
+
+# Database setting
+POSTGRES_HOST_APP=appdb
+POSTGRES_HOST_FCREPO=fcrepodb
+POSTGRES_PORT=5432
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=password
+POSTGRES_DB_APP=hyrax_production
+POSTGRES_DB_TEST=hyrax_test
+POSTGRES_DB_FCREPO=fcrepo
+
+# Redis setting
+REDIS_HOST=redis
+REDIS_PORT=6379
+
+# Fedora setting
+FEDORA_URL_SCHEME=http
+FEDORA_HOST=fcrepo
+FEDORA_PORT=8080
+# FCREPO_ENDPOINT=hyrax_production # Optional - default is production
+FCREPO_PATH=/fcrepo/rest # Optional - default is /rest
```

Comment:
When I tried the pull request, I got the following error.

```
web_1       | ActiveFedora: loading fedora config from /data/config/fedora.yml
web_1       | ActiveFedora: loading solr config from /data/config/solr.yml
web_1       | Fedora URL (http://fcrepo:8080'/fcrepo/rest') does not end with /rest. This could be a problem. Check your fedora.yml config
```

I removed quotations from ``FCREPO_PATH`` from the default env file.